src/components/Courses/tests/courses.test.js

```diff
@@ -0,0 +1,89 @@
+import { render, screen, fireEvent, waitFor } from '@testing-library/react';
+import { Courses } from '../Courses';
+import { Provider } from 'react-redux';
+import configureStore from 'redux-mock-store';
+import { MemoryRouter } from 'react-router-dom';
+import { CourseAdd } from 'components/CourseForm/CourseAdd';
+
+const mockStore = configureStore();
+
+const mockAuthors = [
```

Comment:
When you're using the same mock for several tests, you may benefit from extracting it into a separate mock file and importing it in several tests. Usually you can save these mocks in .json format, then enable JSON imports in your TypeScript config file (tsconfig.json) with
```
"resolveJsonModule": true,
"esModuleInterop": true,
```

Reply:
I created a .json file and imported mock data to test files.Thanks!